schema.sql

```diff
@@ -12,4 +12,63 @@ CREATE TABLE animals (
 /* ADD NEW COLUMN TO TABLE */
 
 ALTER TABLE animals
-ADD COLUMN species VARCHAR(50);
\ No newline at end of file
+ADD COLUMN species VARCHAR(50);
+
+/* Multiple tables */
+
+/* Owners table */
+
+CREATE TABLE owners (
+    id INt GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    full_name VARCHAR(150),
+    age INTEGER
+);
+
+/* Species table */
+CREATE TABLE species (
+    id INT GENERATED ALWAYS AS IDENTITY  PRIMARY KEY,
+    name VARCHAR(150)
+);
+
+/* Modify animals table */
+
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals
+ADD COLUMN species_id INT
+REFERENCES species(id)
+ON DELETE CASCADE;
+
+ALTER TABLE animals
+ADD COLUMN owner_id INT
+REFERENCES owners(id)
+ON DELETE CASCADE;
+
+/* Join Tables */
+
+/* Vets Table */
+
+CREATE TABLE vets (
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name VARCHAR(150),
+    age INTEGER,
+    date_of_graduation date
+);
+
+/* Specialization Table */
+
+CREATE TABLE specialization(
+    vets_id INT REFERENCES vets (id),
+    species_id INT REFERENCES species (id),
+    PRIMARY KEY (vets_id, species_id)
+);
```

Comment:
- [ ] Good job creating the table, Yet It's better to use the plural form when creating a table as it might contain multiple records of `specialization`. please use `specializations` instead. this might sound silly now, but it will save you a lot of trouble in the future 😊.
> Please don't forget to update all your related information, like when you `insert` to the table or execute the `update` statement.

Reply:
Hello @Tadesse-Alemayehu, thank you for taking out time to go through this PR and suggesting this required change. I have gone ahead to implement the change.